query_optimizer/rules/ExtractCommonSubexpression.hpp

```diff
@@ -111,7 +111,7 @@ class ExtractCommonSubexpression : public Rule<physical::Physical> {
   bool visitAndCount(
       const expressions::ExpressionPtr &expression,
       ScalarCounter *counter,
-      ScalarHashable *hashable);
+      ScalarHashable *hashable) const;
```

Comment:
All methods below in this file could be marked as `const`:

* `transformExpressions`
* `transformExpression`
* `visitAndTransform`